charts/meteor-pipelines/templates/import-jupyterhub-image.yaml

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
Also.. thinking about it now, I think we can remove this pipeline resource from here, so we don't pollute this chart. We can use https://github.com/thoth-station/helm-charts/pull/23/files#diff-502c30bc43b0de34014a4c791a3b5480a7666fb05f7346476a447ed04eee1e20R1 instead, once your PR is merged, wdyt @mimotej ? :slightly_smiling_face:

Reply:
Sure I was thinking that it might be little bit rendundant in my PR, will do :+1: